config/fxdata/objects.cfg

```diff
@@ -713,3 +713,13 @@ Properties = CHOWNED_ON_ROOM_CLAIM
 Name = SPELLBOOK_POSS
 Genre = SPELLBOOK
 Properties = CHOWNED_ON_ROOM_CLAIM
+
+[object136]
+Name = NUM_1
+Genre = DECORATION
+Properties = CHOWNED_ON_ROOM_CLAIM DESTROYED_ON_ROOM_PLACE
+ObjectSize = 200
+Image3d = "mbox.tga"
```

Comment:
Why Image/Region and Image3D/Region3D. Both the front view and the top view are in 3D right? In creature.cfg all the stats are separated by adding FP to the first person variants, so like:
Image
Region
FPImage
FPRegion

Would that work here?

Reply:
moved to ImageFP and RegionFP

Reply:
I think less generic should be first because otherwise it would be dozens of FPsomething